config/config_test.go

```diff
@@ -635,6 +635,9 @@ var expectedErrors = []struct {
 	}, {
 		filename: "labeldrop5.bad.yml",
 		errMsg:   "labeldrop action requires only 'regex', and no other fields",
+	}, {
+		filename: "labelmap.bad.yml",
+		errMsg:   "\"l-$1\" is invalid 'replacement' for labelmap action",
```

Comment:
This indicates that this won't even parse (which would be fine too), so the new relabel code wouldn't apply. I don't see code changes implementing this though.

Reply:
Yes, I'd just keep the config.

Reply:
Done.